pdi-dataservice-client/src/main/java/org/pentaho/di/trans/dataservice/jdbc/ThinResultSetMetaData.java

```diff
@@ -98,7 +98,7 @@ public int getColumnType( int column ) throws SQLException {
       case ValueMetaInterface.TYPE_DATE:
         return java.sql.Types.TIMESTAMP;
       case ValueMetaInterface.TYPE_BIGNUMBER:
-        return java.sql.Types.DECIMAL;
+        return java.sql.Types.JAVA_OBJECT;
```

Comment:
JAVA_OBJECT?  That seems wrong.  Mapping BIGNUMBER (treated as BigDecimal) ->DECIMAL  is consistent with kettle.

Do you know why the wrong type is being used for aggregation?  Is this really an issue in Mondrian's type handling? (maybe JdbcDialectImpl.getType)